docs/functions.json

```diff
@@ -1179,12 +1179,12 @@
         ]
     },
     {
-        "signature": "split(text, delimiters)",
-        "summary": "Splits `text` based on the given characters in `delimiters`.",
-        "detail": "Empty values are removed from the returned list.",
+        "signature": "split(text, [,delimiters])",
```

Comment:
There is a family of functions which tokenize text into words and can take a string of delimiter chars, i.e. `split`, `word`, `word_count`, `word_slice`.. figured for consistency `split` could let the delimiters be optional like the rest of them.